Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -48,11 +48,7 @@ def __eq__(self, other):
 		return self.coordinates == other.coordinates and self.axes == other.axes
 
 	def getUsedPoints(self):
-		result = set()
-		for i, point in enumerate(self.coordinates):
-			if point is not None:
-				result.add(i)
-		return result
+		return set([i for i,p in enumerate(self.coordinates) if p is not None])
```

Comment:
you could remove the inner list comprehension `set(i for i, p ...)` works the same, I'd expect it to also be a bit faster

Reply:
I know. And for years I've measured and the generator version is slower. Is till slightly true today :(.